### PR TITLE
update the comments

### PR DESCRIPTION
**Pull request by yorkerlin**

### CODE REVIEW

File: src/shogun/machine/gp/MultiLaplacianInferenceMethod.cpp [hunk 1]
```diff
@@ -362,6 +362,8 @@ void CMultiLaplacianInferenceMethod::update_alpha()
 		m_nlz+=Psi_New;
 	}
 
+	if (Psi_Old-Psi_New>m_tolerance && iter>=m_iter)
+		SG_WARNING("Max iterations attained but the optimizer is not converged\n");
```
Comment (karlnapf) [inline src/shogun/machine/gp/MultiLaplacianInferenceMethod.cpp]: maybe print the difference here?

Comment (karlnapf) [inline src/shogun/machine/gp/MultiLaplacianInferenceMethod.cpp]: and also the m_tolerance level

"Max iterations (%d) reached, but convergence level (%f) is not yet below tolerance (%f)"

**Comment by karlnapf** [comment]
This is good to merge but I suggest to change these warning messages to be more informative

**Comment by yorkerlin** [comment]
@karlnapf 
let me know whether this PR is merge-able

File: src/shogun/machine/gp/MultiLaplacianInferenceMethod.cpp [hunk 2]
```diff
@@ -362,6 +362,10 @@ void CMultiLaplacianInferenceMethod::update_alpha()
 		m_nlz+=Psi_New;
 	}
 
+	if (Psi_Old-Psi_New>m_tolerance && iter>=m_iter)
```
Comment (karlnapf) [inline src/shogun/machine/gp/MultiLaplacianInferenceMethod.cpp]: Shouldn't this b absolute value?

**Comment by karlnapf** [comment]
All good
Double check the abs value of convergence check

**Comment by karlnapf** [comment]
I'll merge when back on my computer, cannot do from a phone

**Comment by karlnapf** [comment]
@iglesias Yeah I know, but I guess there is a point in not merging things on a phone, since it might be "too quickly" ;) I am very inclined to such actions!

**Comment by karlnapf** [comment]
Here we go

**Pull request merged**